sematic/log_reader.py

```diff
@@ -193,6 +193,18 @@ def load_log_lines(
         filter_strings,
     )
     run = object_source.get_run(run_id)
+
+    if run.original_run_id is not None:
+        return LogLineResult(
+            more_before=False,
+            more_after=False,
+            lines=[],
+            continuation_cursor=None,
+            log_unavailable_reason=(
+                f"Run output sourced from original run {run.original_run_id}."
+            ),
+        )
```

Comment:
In this case, why not simply doing
```
run = object_source.get_run(run.original_run_id)
```
and simply adding a warning that we are showing logs from the original run?

Reply:
Done. @augray Please review, as the changes are involved.